src/cron.ts

```diff
@@ -57,11 +63,17 @@ export const getAllRFCRemarks = async (startDate: Date): Promise<{ url: string;
     }
 
     logger.info(`Available referendas: ${query}`);
-    const hashes: { url: string; hash: string }[] = [];
+    const ongoing: OpenReferenda[] = [];
+    const completed: CompletedReferenda[] = [];
+
+    const subsquareApi = new SubsquareApi();
     for (const index of Array.from(Array(query).keys())) {
       logger.info(`Fetching elements ${index + 1}/${query}`);
 
-      const refQuery = (await api.query.fellowshipReferenda.referendumInfoFor(index)).toJSON() as { ongoing?: OnGoing };
+      const refQuery = (await api.query.fellowshipReferenda.referendumInfoFor(index)).toJSON() as {
```

Comment:
It's a little confusing to call it a query, at this point the query was already executed and we have the result info.

How about calling it `refInfo`?